django/inventory_aws_ec2.yml

```diff
@@ -0,0 +1,13 @@
+plugin: aws_ec2
+regions:
+  - us-east-1
+filters:
+  instance-state-name: running
+  tag:owner: anish.sapkota
+  tag:Name: ansible_anish* 
+hostnames:
+  - instance-id
+
+groups:
+  public_server: "'public' in tags.Name"
```

Comment:
look after some unique naming convention method for servers so that they do not conflict with other server names.

Reply:
Added pyenv in server name filter.